data/human/campaign events.txt

```diff
@@ -155,11 +155,87 @@ mission "event: war begins"
 			`	Then, in the early morning, nuclear explosions on Geminus and Martini destroyed the Republic Navy Yard and the Galactic Stock Exchange, crippling the military and economic center of the Republic. The "Free Worlds" disclaim any responsibility for the attacks, despite the timing and the fact that the attacks serve their purposes very well.`
 			`	Unless the terrorists can be found or turn themselves in, civil war is imminent. It has been less than a year since you bought your ship, and already the galaxy is falling apart on you.`
 				decline
+			
+			label geminus
+			action
+				set "bombing witness"
+			`	The moment you step off your ship you realize something is very wrong. Heavily armed Navy personnel flank either side of the spaceport building's exit, and more soldiers are patrolling the port. A liaison officer is herding a group of civilians toward a low bunker beside a nearby landing pad. On the landing pad itself, soldiers are struggling to set up a heavy plasma cannon mounted on an anti-aircraft turret.`
+			`	The civilians safely in the bunker, the liaison officer turns to face you. "I'm sorry, captain, but I'm going to have to ask you to leave the planet. We've got a carrier squadron coming in, and we need all the pads free."`
+			choice
+				`	"Of course."`
+					goto summary
+				`	"What's happening?"`
+				`	"Is this some sort of training exercise?"`
+				`	"What makes you think you can just order me to leave?"`
+					goto noncompliance
+
+			`	The liaison officer sighs. "Captain, I'm afraid the planet is in a state of emergency. Yesterday, some rogue planets down south decided they wanted to secede from the Republic. Sent Parliament an ultimatum, wanted electoral and tax reform, free trade, all the same things they've been complaining about for years. No big deal. Three hours ago, though, the Navy Yard here on Geminus was bombed along with the Galactic Stock Exchange on Martini. We don't know if it's the work of the 'Free Worlds' rebels or not, and I can't promise you there aren't going to be more attacks. All I know is that whoever did this used nuclear weapons, so we've got three thousand injured people we need to get to off-planet medical care immediately."`
+			choice
+				`	"I'll get out of the way, then."`
+					goto end
+				`	"Sorry, I didn't know. I'll be out of your hair as soon as I can lift off."`
+					goto end
+				`	"Is there any way I can help?"`
+					goto help
+
+			label martini
+			action
+				set "bombing witness"
+			`	The moment you step off your ship you realize something is very wrong. Heavily armed Navy personnel flank either side of the spaceport building's exit, and more soldiers are patrolling the port. Sirens are going off along the streets outside, and a liaison officer is herding a group of civilians toward a military transport auxiliary. Still more people are stretched out on cots in a makeshift field hospital in the center of the spaceport.`
```

Comment:
FYI when I added the Martini distress missions I added in the detail that the main spaceport of Martini was destroyed. Might be worth it to change the start to mention you were redirected to a different spaceport on Martini.

It could be changed; atm I think only the post-bombing Martini spaceport description I added actually references the destruction of the spaceport. But that would then be something for another PR.